lecture_3/03_project/calculator/internal/engine/handler/betRepository.go

```diff
@@ -3,11 +3,10 @@ package handler
 import (
 	"context"
 
-	domainmodels "github.com/superbet-group/code-cadets-2021/lecture_3/03_project/controller/internal/domain/models"
+	domainmodels "github.com/superbet-group/code-cadets-2021/lecture_3/03_project/calculator/internal/domain/models"
 )
 
 type BetRepository interface {
 	InsertBet(ctx context.Context, bet domainmodels.Bet) error
-	UpdateBet(ctx context.Context, bet domainmodels.Bet) error
-	GetBetByID(ctx context.Context, id string) (domainmodels.Bet, bool, error)
+	GetBetsBySelectionID(ctx context.Context, id string) ([]domainmodels.Bet, bool, error)
```

Comment:
`exists` flag is not needed when fetching multiple bets, you can just return an empty slice.
Note this doesn't apply to methods where only a single bet is fetched as in that case some kind of a struct is guaranteed to be returned, so you cannot determine whether the empty struct was returned because it doesn't exist, or it was the actual returned value.